core/src/main/java/org/bitcoinj/crypto/DeterministicKey.java

```diff
@@ -496,20 +497,52 @@ else if (outputScriptType == Script.ScriptType.P2WPKH)
         return ser.array();
     }
 
+    /**
```

Comment:
I think `@deprecated` and `@Deprecated` should be enough to mark a symbol as deprecated. No need for another marker in the text.

Reply:
Yeah, I'm not sure why I did that. Will fix.